DESCRIPTION

```diff
@@ -1,9 +1,9 @@
 Package: inline
 Version: 0.3.16.1
-Date: 2020-11-25
+Date: 2020-11-26
```

Comment:
:grinning: 

I usually leave version micro-variants and date for releases. (Also, I just made it 0.3.16.2 here on the checkout of the PR to make it distinct, but I prefer if these are not touched by PRs, so all good.)

Reply:
:+1: